src/lib/spinel/radio_spinel.cpp

```diff
@@ -1071,6 +1071,22 @@ otError RadioSpinel::GetIeeeEui64(uint8_t *aIeeeEui64)
     return OT_ERROR_NONE;
 }
 
+otError RadioSpinel::SetIeeeEui64(const otExtAddress &aIeeeEui64)
+{
+    otExtAddress addr;
+    otError      error;
+
+    for (size_t i = 0; i < sizeof(addr); i++)
+    {
+        addr.m8[i] = aIeeeEui64.m8[sizeof(addr) - 1 - i];
+    }
+
+    SuccessOrExit(error = Set(SPINEL_PROP_VENDOR_NXP_SET_EUI64_CMD, SPINEL_DATATYPE_EUI64_S, addr.m8));
```

Comment:
You use `NXP_SET_` here but the `spinel.h` `SPINEL_PROP_VENDOR_SET_EUI64_CMD` is new one.

Also suggest using a vendor specific spinel property for this, rather a new spinel command.

Reply:
This was a mistake copy/paste → it is SPINEL_PROP_VENDOR_SET_EUI64_CMD with the definition in src/lib/spinel/spinel.h